### ISSUE TITLE
[Worldgen] Apatite vein spawning in Underdark

### ISSUE THREAD

**Issue by richardhendricks** [GTNewHorizons/NewHorizons#2467]
Not sure if a bug or not, but noticed a few Apatite veins spawning in Underdark.  Worldgen.cfg has apatite enabled in Underdark at/near line 9254

**Comment by Dream-Master** [comment]
I think i add it so not a bug.

**Comment by richardhendricks** [comment]
well, it is weird since it is not at the same rate at the other veins.  Maybe make a dedicated custom Deep Dark vein for apatite? Not sure how you want this.

**Comment by Dream-Master** [comment]
did i not add a custom vein?

**Comment by richardhendricks** [comment]
No it is the same vein as the overworld.

**Comment by Dream-Master** [comment]
Maybe i add it because i cant add appatite a second time as first or second Ore in a Mix.

**Comment by richardhendricks** [comment]
Really?  I don't see why. If you want/need me to investigate that issue, I can modify this ticket.

**Comment by Dream-Master** [comment]
The problem is you can set one ore type only once as primary or secondary.  If it set in a mix allready you don't have to set it a third time. The ore and the mix will not appear. I tested it before the massive ore changes and it not works. @richardhendricks would be nice we can set one ore type as many I need it.

**Comment by richardhendricks** [comment]
Yeah that is a strange restriction, I don't recall anything in the code that causes that.  Will update ticket to reflect this as the new goal.

I also want to refactor the orevein code overall, I'll setup a separate ticket for that.
